### PR TITLE
[EventDispatcher] Fix FC layer (bis)

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Component/EventDispatcher/WrappedEvent.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    public function getWrappedEvent()
+    {
+        return $this->event;
+    }
+
+    public function isPropagationStopped()
+    {
+        if (!$this->event instanceof ContractsEvent && !$this->event instanceof StoppableEventInterface) {
+            throw new \BadMethodCallException('Wrapped event is not stoppable.');
```
Comment (nicolas-grekas) [inline src/Symfony/Component/EventDispatcher/WrappedEvent.php]: return false here I think

**Review by nicolas-grekas — COMMENTED** [review]
LGTM with one comment

File: src/Symfony/Component/EventDispatcher/WrappedEvent.php (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Component/EventDispatcher/WrappedEvent.php]: done thanks

**Review by nicolas-grekas — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**